docs/Multiple-Bundle-Stores.md

```diff
@@ -62,6 +62,19 @@ Once a bundle store is created, you can restart the server and set the `CODALAB_
 CODALAB_DEFAULT_BUNDLE_STORE_NAME=blob codalab-service start ...
 ```
 
+## Infer store type from URL
```

Comment:
I don't think we need a new section for this. Can we just modify the previous commands so we remove `--storage-type gcs`, etc. from the commands?